src/ActionModal/components/ActionModal.vue

```diff
@@ -451,10 +455,10 @@ export default {
       ) {
         // hardcoding terra tax here until we have it in the API
         const terraTax = 0.008
-        return (
-          Number(this.gasEstimate) * Number(this.gasPrice) +
-          Number(this.amount) * terraTax
-        )
+        const terraExtraFees = this.isMaxAmount
+          ? 0
+          : Number(this.amount) * terraTax
+        return Number(this.gasEstimate) * Number(this.gasPrice) + terraExtraFees
```

Comment:
will this work? your just not sending the required fees
better solution to calculate the extra fees into max
or just ignoring this for now...

Reply:
Yes, actually I am realizing this is a terrible solution anyway, because what happens if we are sending the max amount minus a little bit, like 0.0000001. 

Then we would still need to pay for the tax * amount and get the same error message...

Ah, this is what you are saying anyway

Reply:
Yeah, I am going to change this to your solution. This is a bad one